src/components/global/Hero/styles.js

```diff
@@ -0,0 +1,69 @@
+export const heroStyles = {
+    heroBackground: {
+        width: '100%',
+        height: { xs: '35vh', md: '70vh' },
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+        backgroundRepeat: 'no-repeat'
+    },
+    heroContainer: {
+        display: 'flex',
+        flexFlow: 'column',
+        justifyContent: 'flex-start',
+        alignItems: { xs: 'center', md: 'flex-start' },
+        width: { xs: '100%', md: '575px' },
+        height: '100%'
+    },
+    heroContainerAlt: {
+        display: 'flex',
+        flexFlow: 'row',
+        alignItems: 'center',
+        alignContent: 'center',
+        width: '100%',
+        height: '100%'
+    },
+    heroRightAlign: {
+        justifyContent: {xs: 'center', md: 'flex-end'},
+    },
+    heroCenterAlign: {
+        justifyContent: 'center',
+    },
+    homeH1: {
+        color: '#ffff',
+        textShadow: '0px 4px 4px #0000004D',
+        lineHeight: { xs: '45px', md: '80px' },
+        flexBasis: { xs: '100px', md: '160px' },
+        fontSize: { xs: '30px', md: '56px' },
+        pt: { xs: '25px', md: '113px' },
+        pl: { xs: 0, md: '56px' },
+        width: { xs: '80%', md: '100%' },
+        textAlign: 'center'
+    },
+    projectsH1: {
```

Comment:
Find this one little tough to read on small screens, we can discuss at meeting.

Reply:
Text should be white to match the rest. It just wasn't updated in Figma. It will also need top padding lowered to stop it from pushing out of the hero on small screens. 

<img width="656" alt="Screenshot 2023-11-05 at 2 32 16 PM" src="https://github.com/codeforpdx/codepdx_website/assets/114430929/758605f0-a195-469a-a41d-9d3596f32c21">

![Screenshot 2023-11-05 at 2 35 09 PM](https://github.com/codeforpdx/codepdx_website/assets/114430929/b6714f67-36ea-47a6-9270-66c5c8009876)

Reply:
text color updated!